Task 1/Trapping Rain Water Problem/Question.md

```diff
@@ -0,0 +1,21 @@
+# Trapping Rain Water Problem
+## Problem Statement
+### Given n non-negative integers representing an elevation map where the width of each bar is 1, compute how much water it can trap after raining.
+
+
+### Input: height = [0,1,0,2,1,0,1,3,2,1,2,1]
+### Output: 6
+### Explanation: The above elevation map (black section) is represented by array [0,1,0,2,1,0,1,3,2,1,2,1]. In this case, 6 units of rain water (blue section) are being trapped.
```

Comment:
You don't have to have the entire markdown in headings. But this is upto you. This won't prevent me from merging this pull request.